### DISCUSSION TITLE
dbt run --select dbt_artifacts on every prod run?

### DISCUSSION THREAD

**Discussion by CMoney88** [brooklyn-data/dbt_artifacts#350] [Q&A]
Hello, we have a production DBT job that runs using DBT cloud every 4 hours. We have setup dbt_artifacts and it is working properly. My question is, does the command `dbt run --select dbt_artifacts` need to be run as part of every DBT cloud job? Or does that only need to be run once? Should it be inline with the existing jobs, or be a stand alone job that runs on it's own schedule.

I don't fully understand the commands behavior and how it should be used. Thanks!

**Comment by TommyD053** [comment]
Hi Cameron, 
if you'll update your dbt_project.yml on-run-end  (https://docs.getdbt.com/reference/project-configs/on-run-start-on-run-end) with 
- "{{ dbt_artifacts.upload_results(results) }}" (https://github.com/brooklyn-data/dbt_artifacts point 1) then the artifacts will be running after each invocation (seed, run, test) of every job.  You won't need to put a command for it. 
Hope that helps :)
